.github/workflows/build-artifact.yaml

```diff
@@ -24,8 +25,10 @@ jobs:
       - uses: ./.github/actions/install-python-deps
       - name: Build tt-metal and libs
         run: |
-          make build PYTHON_ENV=${{ github.workspace }}/python_env
-          make tests
+          cmake -DCMAKE_BUILD_TYPE=ci -DCONFIG=ci -B build
+          cmake --build build --parallel
+          cmake --build build --target tests -- -j4
```

Comment:
does triggering the tests target automatically trigger a build?

Reply:
oops lol yes you're right